### ISSUE TITLE
Escaping characters "<" and ">" in messages with json formatting

### ISSUE THREAD

**Issue by Leomixer17** [bensku/Skript#1190]
As is foreseeable, putting the characters "<" and ">" in tags for json formatting (eg. tooltips), makes it buggy, so you can't use it as text.
Which escape could be used to have the possibility to put "<" and ">" in the text of a tag?

**Comment by TheBentoBox** [comment]
Have you tried a backslash? No idea if it works but that's the standard escape character.

**Comment by bensku** [comment]
Using colors on tooltips is not working reliably at the moment. Gotta fix at some point.

**Comment by Leomixer17** [comment]
@bensku it's not for colors, is simply for putting > and < characters

**Comment by Leomixer17** [comment]
@TheBentoBox Yes, but doesn't work, escaping is not automatic, because in Java the character "\" is automatically replaced by a double slash

**Comment by bensku** [comment]
This is Skript, not Java. I'm actually not sure if this could be an user error...

**Comment by Leomixer17** [comment]
@bensku Which error?

**Comment by Snow-Pyon** [comment]
The user is suggesting escaping the angle brackets since it doesn't work very well currently. For example the following:
```
send "<tooltip:>>>test"
```
will send:
```
>>test # with a space as tooltip
```
instead of:
```
test # with ">" as tooltip
```

**Comment by Tjmaxmillion** [comment]
This issue still persists, Example code below.
```
Command /ttp <text>:
    trigger:
        send formatted "<ttp:%unformatted arg-1%>hover me" 
```

Commands used in order.
/ttp test
/ttp <test

![image](https://user-images.githubusercontent.com/76654305/210384362-40935590-9c3e-409f-bedb-13a9439d54cb.png)
